otx/algorithms/detection/adapters/mmdet/utils/config_utils.py

```diff
@@ -207,64 +197,56 @@ def prepare_for_training(
     return config
 
 
+@check_input_parameters_type()
+def prepare_work_dir(config: Union[Config, ConfigDict]) -> str:
+    """Prepare configs of working directory."""
+    base_work_dir = config.work_dir
+    checkpoint_dirs = glob.glob(os.path.join(base_work_dir, "checkpoints_round_*"))
+    train_round_checkpoint_dir = os.path.join(base_work_dir, f"checkpoints_round_{len(checkpoint_dirs)}")
+    os.makedirs(train_round_checkpoint_dir)
+    logger.info(f"Checkpoints and logs for this training run are stored in {train_round_checkpoint_dir}")
+    config.work_dir = train_round_checkpoint_dir
+    if "meta" not in config.runner:
+        config.runner.meta = ConfigDict()
+    config.runner.meta.exp_name = f"train_round_{len(checkpoint_dirs)}"
+    # Save training config for debugging. It is saved in the checkpoint dir for this training round.
+    # save_config_to_file(config)
+    return train_round_checkpoint_dir
+
+
 @check_input_parameters_type()
 def config_to_string(config: Union[Config, ConfigDict]) -> str:
```

Comment:
If this is for mmcv config, then would it be more propriate to be at mmcv/utils ?

Reply:
From what I've looked at, this is a deprecated function. I will remove it.